ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -45,12 +46,23 @@ def patch():
         _patch(elasticsearch)
 
 
+def _determine_transport_module(elasticsearch):
+    transport_module = getattr(elasticsearch, "transport", False)
+    if not transport_module:
+        import elastic_transport
+
+        transport_module = getattr(elastic_transport, "_transport")
```

Comment:
```suggestion
        transport_module = elastic_transport._transport
```

Reply:
This was integrated in the latest commit, thank you! 👍